classes/DirtOrbit.sc

```diff
@@ -186,6 +186,7 @@ DirtOrbit {
 			~cut = 0.0;
 			~unit = \r;
 			~n = \none; // sample number or note
+			~note = \none; // sample pitch or note
```

Comment:
this would then break the automatic derivation of `note`from `n` if note is not specified (currently, `== nil`, so that `~midinote = #{ ~note ? ~n + (~octave * 12) };` will ues `n`).

Reply:
Ah yes.
Without this line, I'm having some problems working out whether note has been set. For instance this:
```
("Note is " + ~note).postln;
if (~note != \nil) {
  ("Note is not nil.").postln
};
```
produces:
```
Note is  nil
Note is not nil.
```
I'm missing something here I think!

Reply:
try this:

```
("Note is " + ~note).postln;
if (~note != nil) {
  ("Note is not nil.").postln
};
```

or,

```
("Note is " + ~note).postln;
if (~note.notNil) {
  ("Note is not nil.").postln
};
```

Reply:
That worked, thanks!